### PR TITLE
Update eip-2981.md

### PR DESCRIPTION
**Pull request by lenifoti**
Proposed changes:
Remove/modify statements that imply that this is an extension of ERC-721.  There is no hard dependency.
Add a note to the effect that it is not exclusive to ERC-721.
Add requirement that marketplaces should not accept royalty amounts where the NFT uses the extra data, unless they are operating under an extended EIP.
Rename _INTERFACE_ID_ERC721ROYALTIES to _INTERFACE_ID_ERC2981, per the pattern used by ERC721.
Extra wording around the unenforcability of royalties to stress that even enforcing in code can be defeated.
Re-write the backward compatibility section.  I think this predated our cut-down version and there are no backward compatibility concerns other than 
- existing NFTs obviously cannot take advantage of it and 
- marketplaces will need to accommodate this EIP in addition to their current strategies.

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - eip-2981.md requires approval from one of (vexycats, jamesmorgan, blmalone, seibelj)
 @vexycats @jamesmorgan @blmalone @seibelj

File: EIPS/eip-2981.md
Comment (lenifoti) [inline EIPS/eip-2981.md:66]: Need to fix -> _INTERFACE_ID_ERC2981

File: EIPS/eip-2981.md
Comment (lenifoti) [inline EIPS/eip-2981.md:146]: NFTs that support this standard must implement ERC-165 to allow introspection, ...

File: EIPS/eip-2981.md
Comment (cliffhall) [inline EIPS/eip-2981.md:3]: Perhaps **NFT Royalty Standard**  so as to remove the 721 specificity?

Comment (wighawag) [reply EIPS/eip-2981.md:3]: I would not use NFT as ERC1155 allows tokens to not be NFT

Comment (cliffhall) [reply EIPS/eip-2981.md:3]: The goal is for NFT creators to receive royalties, correct? Saying ERC-721 in the title seems overly specific since 1155 should be supported as well. But **ERC-721 / ERC-1155 Royalty Standard** would be doubling down on specificity because a new NFT EIP would not be indicated. The fact that 1155 is technically semi-fungible seems less important than clarity in the title, IMHO.

Comment (lenifoti) [reply EIPS/eip-2981.md]: I agree that we want to widen the scope.  Leading with ERC721 is both restrictive and misleading.  I'm torn between something as wide as  "Simple Royalty Standard" and something that is a bit more specific about what problem this solves: "Royalty signaling Standard".  That leaves it open for a "Royalty payment standard" etc.

**Review by lenifoti — COMMENTED** [review]
reviewed

**Review by lenifoti — COMMENTED** [review]
Reviewed

File: EIPS/eip-2981.md
Comment (cliffhall) [inline EIPS/eip-2981.md:19]: > This standard complements the ERC-721 specification and ERC-1155 specification by enabling...

**Reduce verbosity:**
This standard complements the ERC-721 and ERC-1155 specifications by enabling...

File: EIPS/eip-2981.md
Comment (cliffhall) [inline EIPS/eip-2981.md:26]: > This standard is intended as a complement to the ERC-721 specification and ERC-1155 specification allowing the...

**Reduce verbosity:**
This standard is intended as a complement to the ERC-721 and ERC-1155 specifications allowing the...

**Review by cliffhall — COMMENTED** [review]
Title is excellent. Inclusion of 1155 alongside 721 throughout the text makes it clear both standards are supported.

File: EIPS/eip-2981.md
Comment (axic) [inline EIPS/eip-2981.md:34]:
```suggestion
While, ERC-721 NFTs are the primary use case at this time, we envision that the functionality can be applied to all extensions to ERC721 and other contract interfaces that wish to derive royalties in this way. 
```

Comment (cliffhall) [reply EIPS/eip-2981.md:34]: You'd also want to hyphenate the second instance in that same paragraph.

**Comment by poojaranjan** [comment]
EIP-2981 has acquired "Final" status. Should this pull request be closed?

**Review by jamesmorgan — APPROVED** [review]

**Comment by lightclient** [comment]
The EIP is final now. In the future we should try to ensure that outstanding PRs are addressed before merging something as final. I think I gave the approval for this one, so that's my mistake here. I apologize for that.

**Comment by lightclient** [comment]
(Non-normative changes are still accepted if the author accepts, please reopen if you would like to continue pursuing)

**Pull request closed without merging**
